components/cms/Toolbar.tsx

```diff
@@ -27,15 +27,25 @@ export const Toolbar = styled(({ ...styleProps }) => {
           <CreateContentMenu />
         </Create>
         <Github>
-          {git.all().length > 0 && git.all().map((git: any) => git.component())}
+          {git.all().length > 0 &&
+            git.all().map((git: any, i) => {
+              const id = `${i}-${git.name}`
+              return git.component(id)
+            })}
         </Github>
         <Status>
           {status.all().length > 0 &&
-            status.all().map((status: any) => status.component())}
+            status.all().map((status: any, i) => {
+              const id = `${i}-${status.name}`
+              return status.component(id)
```

Comment:
Maybe try this:
```suggestion
              return <status.component key={id} />
```
then the plugin components don't need to set the key themselves

Reply:
nice!

Reply:
Thank you :)